### PR TITLE
fix: #454, no `file_names` when sync from app

### PR DESCRIPTION
**Pull request by ben-29**

### CODE REVIEW

**Comment by agassiyzh** [comment]
感谢，周末太忙没来得及改

File: scripts/generator/__init__.py [hunk 1]
```diff
@@ -108,7 +108,8 @@ def sync_from_app(self, app_tracks):
                 sys.stdout.write("+")
             else:
                 sys.stdout.write(".")
-            synced_files.extend(t.file_names)
+            if t.__contains__("file_names"):
+                synced_files.extend(t.file_names)
```
Comment (yihong0618) [inline scripts/generator/__init__.py:112]: nice

Comment (frostming) [reply scripts/generator/__init__.py:112]: 应该用 `a in b`

Comment (ben-29) [reply scripts/generator/__init__.py]: Done

**Pull request merged**